src/Atmos/Model/moisture.jl

```diff
@@ -76,25 +78,24 @@ function gradvariables!(::EquilMoist, transform::Vars, state::Vars, aux::Vars, t
 end
 
 
-function diffusive!(::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ν::Union{Real,AbstractMatrix})
+function diffusive!(m::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ν::Union{Real,AbstractMatrix})
   # turbulent Prandtl number
   diag_ν = ν isa Real ? ν : diag(ν) # either a scalar or vector
   D_q_vap = D_q_liq = D_q_ice = D_q_tot = diag_ν / Prandtl_t # either a scalar or vector
-  
+
   # diffusive flux of q_tot
-  # FIXME 
+  # FIXME
   ρd_q_vap = state.ρ * (-D_q_vap) .* ∇transform.moisture.q_vap # a vector
   ρd_q_liq = state.ρ * (-D_q_liq) .* ∇transform.moisture.q_liq # a vector
   ρd_q_ice = state.ρ * (-D_q_ice) .* ∇transform.moisture.q_ice # a vector
 
   diffusive.moisture.ρd_q_tot = ρd_q_vap + ρd_q_liq + ρd_q_ice
 
   D_T = diag_ν / Prandtl_t
-  phase = PhaseEquil(aux.moisture.e_int, q_tot, ρ, aux.moisture.temperature)
-  q = PhasePartition(phase)
+  phase = thermo_state(m, state, aux)
 
   # J is the conductive or SGS turbulent flux of sensible heat per unit mass
-  ρJ = state.ρ * cv_m(q) * D_T * ∇transform.moisture.temperature
+  ρJ = state.ρ * cv_m(phase) * D_T * ∇transform.moisture.temperature
```

Comment:
This isn't right. I am currently updating docs. The way to do this is to compute sum `ρJ + ρD` by diffusing specific total energy `e_tot`. We don't need the fluxes separately.